Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -242,9 +242,10 @@ bool TMemCheck::Action(DebugInterface* debug_interface, u32 value, u32 addr, boo
   {
     if (log_on_hit)
     {
-      NOTICE_LOG(MEMMAP, "MBP %08x (%s) %s%i %0*x at %08x (%s)", pc,
+      NOTICE_LOG(MEMMAP, "MBP %08x (%s) %s%zu %0*x at %08x (%s)", pc,
                  debug_interface->GetDescription(pc).c_str(), write ? "Write" : "Read", size * 8,
-                 size * 2, value, addr, debug_interface->GetDescription(addr).c_str());
+                 static_cast<int>(size * 2), value, addr,
```

Comment:
The `%0*x` format string consumes two arguments: the first, `value`, must be an integer, and only the second, `size * 2`, can be formatted as a specific type.

According to printf(3):
> Instead of a decimal digit string one may write "*" ... to specify that the field width is given in the next argument ... which must be of type int.